Document for imageSizes #188 (#200)
enonic/lib-guillotine: docs/api.adoc

|imageWidths: [Int]
|Generates image URLs for given widths which will be used in the `srcset` attribute of `img` tag.

|image:images/v-530.svg[Since version,opts=inline] imageSizes: String

> |Specifies the width for an image depending on browser dimensions. The value has the following format: `(media-condition) width`. Multiple sizes are comma-separated.

Reviewer: ```suggestion
Specifies image width for a specific browser resolution in the following format: `(media-condition) width`. Multiple sizes are comma-separated.
```

Author: done
